recipes/glow/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "glow" %}
+{% set pypi_name = "glow.py" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
```

Comment:
Why not call it `glow.py` as in the PyPI name?

Reply:
We prefer to use the name `glow`. We only used `glow.py` in pypi because `glow` was taken. The maven artifact, which people must install as well for use on a cluster, is called `glow`: https://mvnrepository.com/artifact/io.projectglow

Reply:
That is fine, particularly if you are involved with this package upstream. However, bare in mind that some users may have a bad experience. One workaround is to create a meta-package named `glow.py` that installs `glow`. But that can be done later in the feedstock if your team agrees.

Reply:
Oh, that's a good idea. We will do that. Thanks!

Reply:
@ocefpaf when you say "in the feedstock", do you mean in the automatically created repo? Do I not need to create a new feedstock for a `glow.py` metapackage?